src/components/NavBar/NavBar.module.css

```diff
@@ -5,34 +5,46 @@
 .container {
   margin-inline: auto;
   max-width: 1440px;
-  padding: 1.5rem;
+  padding: 1.2rem 1rem;
   position: relative;
 }
```

Comment:
I'd recommend adding a z-index here so you can avoid this small bug


https://user-images.githubusercontent.com/49324987/165765309-40bc70a7-99f0-4741-861b-ac7479388bc2.mov


https://user-images.githubusercontent.com/49324987/165765309-40bc70a7-99f0-4741-861b-ac7479388bc2.mov



https://user-images.githubusercontent.com/49324987/165765149-c0ae574f-d54c-4acc-8ced-8b8e2e0c4f54.mov